doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -26,33 +26,45 @@ The main command to provide custom notations for tactics is :cmd:`Tactic Notatio
 Notations
 ---------
 
+
 Basic notations
 ~~~~~~~~~~~~~~~
 
-.. cmd:: Notation
+.. cmd:: Notation @string := @one_term {? ( {+, @syntax_modifier } ) } {? : @scope_name }
+
+   Defines a *notation*, an alternate syntax for entering or displaying
+   a specific term or term pattern.
+
+   This command supports the :attr:`local` attribute, which limits its effect to the
+   current module.
+   If the command is inside a section, its effect is limited to the section.
+
+   Specifying :token:`scope_name` associates the notation with that scope.  Otherwise
+   it is a *lonely notation*, that is, not associated with a scope.
+
+   .. todo PR Are lonely notations not encouraged or perhaps should be removed?
```

Comment:
They are certainly discouraged in libraries, but are definitely convenient in small projects, so no they shouldn't be removed.